weather.py

```diff
@@ -0,0 +1,18 @@
+import requests
+
+GEOCODING_URL = 'https://geocoding-api.open-meteo.com/v1/'
+OPEN_METEO_URL = 'https://api.open-meteo.com/v1/'
+
+city_name = input("Enter a city name: ")
+city_coordinates = requests.get(f"{GEOCODING_URL}search?name={city_name}&count=1")
+
+latitude = float(city_coordinates.json().get('results')[0]['latitude'])
```

Comment:
Your code breaks here if a non-existing city is entered. It is better to handle this